tests/topotests/high_ecmp/r2/frr.conf

```diff
@@ -0,0 +1,266 @@
+int lo
+ ip addr 1.1.1.2/32
```

Comment:
Can we also replace this with a non-Cloudflare IP?

Reply:
done